lib/common/subredditPost.dart

```diff
@@ -98,6 +138,22 @@ class SubredditPost extends StatelessWidget {
     );
   }
 
+  Future<void> _upvote() async {
```

Comment:
Both of these functions should be return types of `Future<void>`, also, can we do camelCase on the naming for these functions?

`_upVote`

Reply:
Hi, I have just changed the return type and the name of the function.
If you decide to accept my PR, may I ask you to add the ["hacktoberfest" topic to this repository](https://hacktoberfest.digitalocean.com/hacktoberfest-update) so that I can participate in the Hacktoberfest? Thank you!

Reply:
@lucacorbucci added the topic, am I fine to merge this branch? or is there any thing else I need to do in terms of the hacktoberfest.